### PR TITLE
3769 UI show fingerprinters as plugins

### PR DESCRIPTION
**Pull request by ilija-lazoroski**
# What does this PR do?

Fixes part of #3769.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [x] Was the documentation framework updated to reflect the changes?
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [x] Added relevant unit tests?
* [x] Do all unit tests pass?
* [x] Do all end-to-end tests pass?
* [x] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [x] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

**Review by cakekoa — COMMENTED** [review]
We're adding some tech debt here, but considering that we're planning on reworking the UI (and thus labeling it all 'tech debt'), minimal harm is done here. We'll work on a better general solution when we refactor the UI

**Comment by VakarisZ** [comment]
UI crashes for me when I try to use it

**Review by VakarisZ — CHANGES_REQUESTED** [review]
Fix or remove the fingerprinter UI

**Comment by VakarisZ** [comment]
Removed UI component for fingerprinters altogether. If we have to use it until UI refactoring, we can 1. export 2. modify them 3. import

File: monkey/common/agent_configuration/agent_sub_configurations.py [hunk 1]
```diff
@@ -172,7 +172,7 @@ class NetworkScanConfiguration(MutableInfectionMonkeyBaseModel):
     tcp: TCPScanConfiguration = Field(
         title="TCP scanner", description="Configure TCP scanning options"
     )
-    fingerprinters: Tuple[PluginConfiguration, ...] = Field(
+    fingerprinters: Dict[str, Dict] = Field(
```
Comment (VakarisZ) [inline monkey/common/agent_configuration/agent_sub_configurations.py:175]: Wrong type hint

**Pull request merged**